universal-login-relayer/lib/core/services/DeploymentHandler.ts

```diff
@@ -1,12 +1,11 @@
 import Deployment from '../models/Deployment';
-import WalletService from '../../integration/ethereum/WalletService';
-import {calculateDeployHash, DeviceInfo, DeployArgs} from '@universal-login/commons';
+import {calculateDeployHash, DeviceInfo, DeployArgs, DeploymentStatus} from '@universal-login/commons';
 import IRepository from './messages/IRepository';
 import {IExecutionQueue} from './messages/IExecutionQueue';
 
 class DeploymentHandler {
   constructor(
-    private walletService: WalletService,
+    _: any,
```

Comment:
Will remove unused parameter in follow up PR

Reply:
ok